cmr-exchange/service-bridge/Dockerfile

```diff
@@ -1,2 +1,9 @@
-FROM openjdk:8u162-jre-slim
-MAINTAINER <http://cmr.earthdata.nasa.gov/>
+FROM clojure
+
+COPY target/cmr-service-bridge-*-SNAPSHOT-standalone.jar /app/cmr-standalone.jar
+COPY resources/ /app/resources/
+EXPOSE 3013
+
+WORKDIR /app
+
+CMD "java" "-jar" "cmr-standalone.jar"
```

Comment:
I think we've left out the command in our other Dockerfiles, not sure if we want to take this out for consistency with the other apps.